package.json

```diff
@@ -42,7 +42,7 @@
     "test": "vitest run",
     "tsc": "tsc",
     "postinstall": "tar -xzC public -f tradingview/tradingview.tgz",
-    "prepare": "husky",
+    "prepare": "ts-patch install && typia patch && husky",
```

Comment:
I honestly don't know what this is for, the typia installer helper did this.

Reply:
```
----------------------------------------
 Typia Setup Wizard
----------------------------------------
Since TypeScript v5.3 update, "tsc" no more parses JSDoc comments.

Therefore, "typia" revives the JSDoc parsing feature by patching "tsc".

This is a temporary feature of "typia", and it would be removed when "ts-patch" being updated.
```

This is the message I got from installing

Reply:
the grammar is great

Reply:
lmao I have no clue what that even means